changelog.d/pa-3054.added

```diff
@@ -0,0 +1,32 @@
+Try-catch-else-finally is now supported in taint analysis.
+
+This kind of statement is supported in languages including Python, Ruby, and Julia.
+
+Python example:
+
+```py
+def f(tainted_input):
+  try:
+    a = 0
+    b = 0
+    c = tainted_input
+    d = tainted_input
+  except RuntimeError:
+    a = tainted_input
+    c = sanitize(c)
+  else:
+    b = tainted_input
+  finally:
+    d = sanitize(d)
+
+  # a is not tainted because exception wasn't thrown
```

Comment:
For any arbitrary code, how do you know that no exception was thrown?

Reply:
If there is a potential path for `a` to get tainted (and that seems to be the case) then it should be tainted, since it's a may-analysis. But I also don't clearly see why this `a` is not tainted here based on the code, need to check the tests you wrote too.

Reply:
Could you point me where in the code we ensure that `a` he isn't tainted?

Reply:
Based on your other comment, I believe you found the code, but happy to discuss more if you have any questions.

I've added a bit more details in the changelog to make it clearer what assumptions are updated in this change.